### ISSUE TITLE
String char_at function

### ISSUE THREAD

**Issue by Alex-Costea** [Alex-Costea/Wiles#17]
Once ticket #6 is done, there will be no way to get the character of a text at a location. This should be fixed, either by implementing .get on strings or creating a new function.

**Comment by Alex-Costea** [comment]
This can be done using `text.as_list.get(index)`. As such, it's not a priority to implement a specific function for it at this time.
